README.md

```diff
@@ -238,10 +238,13 @@ kover {
     isDisabled.set(false) // true to disable instrumentation and all Kover tasks in this project
     engine.set(DefaultIntellijEngine) // to change engine, use kotlinx.kover.api.IntellijEngine("xxx") or kotlinx.kover.api.JacocoEngine("xxx")
     filters { // common filters for all default Kover tasks
-        classes { // common class filter for all default Kover tasks 
+        classes { // common class filter for all default Kover tasks in this project
             includes += "com.example.*" // class inclusion rules
             excludes += listOf("com.example.subpackage.*") // class exclusion rules
         }
+        annotations { // common annotation filter for all default Kover tasks in this project
+            excludes += listOf("com.example.Annotation", "*Generated") // exclude classes and functions marked by specified annotations
```

Comment:
I also wonder if we have to treat `Generated` as excluded by default, with the only option to override it via `includes`? WDYT? How JaCoCo works here?

Reply:
JaCoCo always filters only by the `"*Generated*"` annotation.
I propose at the first stage to let the user specify all annotations with which declarations are marked that they would like to exclude.

If there will be a high request from users that the `"*Generated*"` annotation should be used by default, then we can add a flag to add the `"*Generated*"` to the list (and by default it will be true).